CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+Contributing
+------------
+
+1. [How to contribute](#how-to-contribute)
+    * [Reporting a bug](#reporting-a-bug)
+    * [Writing a pull request](#writing-a-pull-request)
+2. [Writing a component](#writing-a-component)
+    * [Component declaration](#component-declaration)
+    * [Building an example](#building-an-example)
+    * [Write some Tests](#write-some-tests)
+3. [Documentation](#documentation)
+
+## How to contribute
+
+First of all, if you read this, you may want to contribute, so thank you. It's a long and hard work and any help is welcome.
```

Comment:
Try:
"If you are reading this you probably want to contribute, so thanks! Maintaining an open-source project is hard work and any help is welcome."